docs/source/general/registry.rst

```diff
@@ -0,0 +1,87 @@
+########
+Registry
+########
+
+.. _registry:
+
+********************
+Available Registries
+********************
+
+Registries are Flash internal key-value database to store a mapping between a name and a function.
+
+In simple words, they are just advanced dictionary storing a function from a key string.
+
+Registries help organize code and make the functions accessible all across the ``Flash`` codebase.
```

Comment:
maybe add:

If you are creating a custom backbone and want to easily be able to use it in your flash task by simply providing it as a string, you should add it to a registry. You can do that in a few simple steps:

1. import... 
2. init registry
3. Add your function as decortaor or directly
4. you can now access your function from your task!

(add snippets for each step ofcourse)